logging/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Create a library called "Hello" which includes the source file "hello.cxx".
+# The extension is already found. Any number of sources could be listed here.
+add_library (TimeFormatter TimeFormatter.cpp)
+add_library (FileAndFolderHandler FileAndFolderHandler.cpp)
+add_library (Logging Logger.cpp)
+
+# Make sure the compiler can find include files for our Hello library
+# when other libraries or executables link to Hello
+target_include_directories (TimeFormatter PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories (FileAndFolderHandler PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories (Logging PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Why is there 3 different libraries just for the logging ?
That sounds excessive

Reply:
Because you said that it would be nice to split up all the code ^^
I will change that now